manifests/p/pierogii/EATEOF/6932a/pierogii.EATEOF.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: pierogii.EATEOF
+PackageVersion: 6932a
+PackageLocale: en-US
+Publisher: pierogii
+PackageName: Everywhere At The End Of Funk
+License: Apache 2 w/ additional terms and condidions
+ShortDescription: Step into the shoes of a rapper experiencing the horrors of dementia to see how it is really like.
+InstallationNotes: Content Warning: Contains themes of loss, dementia, death, losing memories of a girlfriend, and disturbing music.

```

```suggestion
InstallationNotes: Content Warning - Contains themes of loss, dementia, death, losing memories of a girlfriend, and disturbing music.
```

Given that there needs to be a content warning, I don't know that this package would be appropriate for winget until support for content ratings is implemented

---

InstallationNotes are the closest we get to content warnings, for example when a program malfunctions when you do this you specify it in the installation notes.